### DISCUSSION TITLE
Windows & monochromatic systray icon - use black icon when light theme is active

### DISCUSSION THREAD

**Discussion by okms** [mullvad/mullvadvpn-app#8955] [Feature requests & Ideas]
* mullvad 2020.2 and Windows 10 build 18363
* The *monochromatic tray icon* is currently only white, no matter if you use dark or light theme in windows. It would look much better if the setting used a black icon when the light theme is active. 

Screenshot
![image](https://user-images.githubusercontent.com/170147/74568342-0cf0d100-4f78-11ea-9005-e585a33d67fe.png)

Windows "light" theme is on 
![image](https://user-images.githubusercontent.com/170147/74568501-6953f080-4f78-11ea-90a0-3729aa5521fa.png)

**Comment by faern** [comment]
The goal was never to make it perfect for every theme, but to simply offer the option of not having it only green. And since the default theme is dark, we went with white as the monochromatic version.
We might improve in the future to support both black and white icons, but we can't prioritize it very high at the moment.
